unixfs.go

```diff
@@ -182,6 +182,21 @@ func NewFSNode(dataType pb.Data_DataType) *FSNode {
 	return n
 }
 
+// Filesize gets filesize of format
+func (n *FSNode) Filesize() uint64 {
```

Comment:
This function already exists below.

Reply:
@schomatis I had fixed it. Thx for your reply.

Reply:
@schomatis I am not sure whether we should delete `FromBytes()` in this pr.

Reply:
> @schomatis I am not sure whether we should delete `FromBytes()` in this pr.

No, it's still used in many other repos, but we should probably add a "deprecated" sign or a note recommending the use of `FSNodeFromBytes`.